### PR TITLE
BaSP-MR-54: innitial setup router-shared

### PR DESCRIPTION
**Pull request by IgnacioGalceran**
Innitial setup with all the configurations of routes and shared components.
React router dom version 5.3.0 installed. Layout.jsx changed. Now, every component has their own path.
Shared folder created. Inside, there are every shared components that we need to use in the app. However, we'll see if we need anothers shared components in the future.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "mongoose": "^7.2.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^5.3.0",
```
Comment (FontanR) [inline package.json]: Maybe get rid of "^" symbol in react router dependency so it wont install an updated version

**Review by FontanR — APPROVED** [review]

File: package-lock.json [hunk 2]
```diff
@@ -11,8 +11,10 @@
         "@testing-library/jest-dom": "^5.16.5",
         "@testing-library/react": "^13.4.0",
         "@testing-library/user-event": "^14.4.3",
+        "mongoose": "^7.2.1",
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
+        "react-router-dom": "^5.3.0",
```
Comment (santimenendez98) [inline package-lock.json]: In this part "^" should not go either, when executing "npm i" it gives an error

**Review by jramire5 — APPROVED** [review]
Checked the code and checked in my computer, everything looks OK

File: src/Components/Shared/Button/Button.jsx
Comment (WalterArielPavicich) [inline src/Components/Shared/Button/Button.jsx:1]: I think the file name should be index.jsx

File: src/Components/Shared/Input/Input.jsx
Comment (WalterArielPavicich) [inline src/Components/Shared/Input/Input.jsx:1]: the same in this file, I think it sholud be index.jsx

**Review by WalterArielPavicich — APPROVED** [review]

**Review by santimenendez98 — APPROVED** [review]

**Pull request merged**